src/commands/contract/finders.ts

```diff
@@ -17,10 +17,10 @@ export const ANTELOPE_CLASS_MAPPINGS = {
 export const ANTELOPE_CLASS_WITHOUT_TYPES = ['BlockTimestamp', 'TimePointSec']
 
 export function findTypeFromAlias(typeString: string, abi: ABI.Def): string | undefined {
-    const {type: typeStringWithoutDecorator, decorator} = extractDecorator(typeString)
+    const {type: typeStringWithoutDecorator} = extractDecorator(typeString)
     const alias = abi.types.find((type) => type.new_type_name === typeStringWithoutDecorator)
 
-    return alias?.type && `${alias?.type}${decorator || ''}`
```

Comment:
I'm not sure about this 🤔 Wouldn't we want to either return the decorator here or use it to find the type on line 21? Do you have an example of some code that doesn't work with the current code?